src/matcha_ml/cli/cli.py

```diff
@@ -4,7 +4,6 @@
 import typer
 
 from matcha_ml import __version__
-from matcha_ml.cli import analytics
```

Comment:
We are. The functions are being moved to `core.py`, similar to the way Oscar implemented `get`, and the cli logic has been kept within `cli.py`. This will make the API easier to implement in the future.